pkg/agent/endpoints/peertracker.go

```diff
@@ -10,11 +10,11 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-type peerTrackerAttestor struct {
+type PeerTrackerAttestor struct {
```

Comment:
This requires updating the name in the `endpoints` tests, please run `make test`.